### PR TITLE
Added some awesome presentations from GTAC

### PR DESCRIPTION
**Pull request by upgundecha**
There is more and can be added from https://developers.google.com/google-test-automation-conference

Links are broken from 2011 on GTAC site. However, videos can be found from YouTube Channel https://www.youtube.com/channel/UCtXKDgv1AVoG88PLl8nGXmw

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,6 +276,7 @@ Happy Learning!
 * [Mobile Automation in Continuous Delivery Pipeline](https://www.youtube.com/watch?v=NWDByCVThfE)
 * [Distributed Automation Using Selenium Grid / AWS / Autoscaling](https://www.youtube.com/watch?v=cbIfU1fvGeo)
 * [AWS re:Invent 2017: Use Amazon EC2 Systems Manager to Perform Automated Resilience Testing](https://www.youtube.com/watch?v=jihhLckop6A)
+* [Mobile Automation in Continuous Delivery Pipeline, GTAC 2013](http://www.youtube.com/watch?v=NWDByCVThfE&list=SPSIUOFhnxEiCODb8XQB-RUQ0RGNZ2yW7d), [Slides](http://goo.gl/pOSkp)
```
Comment (abhivaikar) [inline README.md:279]: This is a duplicate. Can you keep one of them?

File: README.md [hunk 2]
```diff
@@ -288,13 +312,26 @@ Happy Learning!
 * [GTAC 2014: Never Send a Human to do a Machine’s Job: How Facebook uses bots to manage tests](https://www.youtube.com/watch?v=_5Sr4EYH7M8)
 * [Facebook Infer: Advanced Static Analysis at Speed and Scale - Facebook - DevTools@Scale2017](https://www.youtube.com/watch?v=gZ3dA4QD8jk)
 * [Building Mobile Apps with Open Source Tools - Simon Stewart - Mobile @ Scale](https://www.youtube.com/watch?v=c2Q22KDjXmI)
+* [Never Send a Human to do a Machine’s Job: How Facebook uses bots to manage tests, GTAC 2014](http://www.youtube.com/watch?v=_5Sr4EYH7M8&list=PLSIUOFhnxEiDFckNDSjKWqOCtd8ksJrh4&index=1), [Slides](http://goo.gl/QtUNxY)
```
Comment (abhivaikar) [inline README.md:315]: Both entries are duplicate. Can you keep one of those?

File: README.md (hunk 1 above)
Comment (upgundecha) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (upgundecha) [reply README.md]: Fixed

**Pull request merged**